tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -721,21 +721,27 @@ hash/test_generic_hash.py::test_ecmp_and_lag_hash:
 
 hash/test_generic_hash.py::test_lag_member_flap:
   skip:
-    reason: 'On Mellanox SPC1 platforms, due to HW limitation, it would not support CRC_CCITT algorithm'
+    reason: 'On Mellanox SPC1 platforms, due to HW limitation, it would not support CRC_CCITT algorithm, for other platforms, skipping until the issue is addressed'
```

Comment:
Can you please clarify the reason a bit more. I understand it is skipped on Mellanox SPC1. The new comment says it is skipped on other platforms while the skip here is only related to `vs`.